crates/nu-std/README.md

```diff
@@ -3,36 +3,78 @@
   <img src="https://media.giphy.com/media/hvRJCLFzcasrR4ia7z/giphy.gif" width="28"></img>
 </h1>
 
-The standard library is a pure-`nushell` collection of commands to allow anyone to build
-complex applications using standardized tools gathered incrementally.
+The standard library is a pure-`nushell` collection of custom commands which 
+provide interactive utilities and building blocks for users writing casual scripts or complex applications.
 
-In this library, you might find `rust`-like `assert` commands to write tests, tools to
-manipulate paths and strings, etc, etc, ...
+To see what's here:
+```
+〉use std
+〉help commands | select name usage | where name =~ "std "
```

Comment:
i think this will be tideous to maintain and to scale...

for the time being, i'd leave the command which is great but remove the big table :+1:

Reply:
I'll telescope the table, but want to show some sample results.  If user wants to see the whole list in the current version, s/he can run the command then.

Reply:
that sounds a great compromise :+1: